muxer_test.go

```diff
@@ -52,11 +53,18 @@ func TestMuxer_generatePAT(t *testing.T) {
 	assert.Equal(t, MpegTsPacketSize, muxer.patBytes.Len())
 	assert.Equal(t, patExpectedBytes(0, 0), muxer.patBytes.Bytes())
 
-	// to check version number increment
+	// Version number shouldn't change
```

Comment:
It would be great to do the same test for PMT. Otherwise the code looks good :)

Reply:
you're right I've added the same test for PMT 👍

Reply:
I think now it's ready to merge :)

Reply:
Oh, you've done it already :)